etc/module.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="../../../../lib/internal/Magento/Framework/Module/etc/module.xsd">
-    <module name="VinaiKopp_EavOptionSetup" setup_version="0.1.0">
+    <module name="VinaiKopp_EavOptionSetup" setup_version="1.0.0">
```

Comment:
Here I bumped the version to attach a tag to use with toran or satis proxy. It would be better if we use a different version from 0.1.0 since you already used this tag. Or maybe do you prefer to overwrite also the tag?